sdc/crypto/test/test_jwt_helper.py

```diff
@@ -52,6 +54,17 @@ def setUp(self):
                 "709eb42cfee5570058ce0711f730bfbb7d4c8ade": {'purpose': KEY_PURPOSE_AUTHENTICATION,
                                                              'type': 'public',
                                                              'value': TEST_DO_NOT_USE_UPSTREAM_PUBLIC_PEM},
+                "EQ_USER_AUTHENTICATION_EQ_KEY": {'purpose': KEY_PURPOSE_AUTHENTICATION,
+                                                  'type': 'private',
+                                                  'value': TEST_DO_NOT_USE_EQ_PRIVATE_KEY},
+
+            }
+        })
+        self.key_store_secondary = KeyStore({
+            "keys": {
+                "EQ_USER_AUTHENTICATION_EQ_KEY": {'purpose': KEY_PURPOSE_AUTHENTICATION,
+                                                  'type': 'public',
+                                                  'value': TEST_DO_NOT_USE_EQ_PUBLIC_KEY},
```

Comment:
If you made the key identifiers `EQ_PRIVATE_KEY` and `EQ_PUBLIC_KEY` then I don't think you'd need the second key store?